### DISCUSSION TITLE
Are the Open Sourcerer and Heart On Your Sleeve badges still being tested?

### DISCUSSION THREAD

**Discussion by ghost** [Schweinepriester/github-profile-achievements#951] [Q&A] [answered]
Hi everyone! 🙏

I wanted to know if anyone has updates on the Open Sourcerer and Heart On Your Sleeve badges. The last I heard, they were still in the testing phase back in 2023. Is there any new information or possible ways to achieve these badges now?

Thanks in advance for your help!

**Comment by Schweinepriester** [comment, marked as answer]
As far as I know GitHub still has not enabled them to be earned publicly…

**Reply by ghost** [reply]
> As far as I know GitHub still has not enabled them to be earned publicly…

I don’t know either; I’ve been waiting for the past two years 🤣 I’ve tried every method, but none of them have worked.😭🙏

**Comment by 23Edu4rd0** [comment]
Honestly, I feel like they'll give me the badge the day I stop contributing to open source (i will do it one day).
